src/pages/dataTableUtilities/actions.js

```diff
@@ -166,3 +166,65 @@ export const deleteTransactionsById = () => (dispatch, getState) => {
 export const refreshData = () => ({
   type: 'refreshData',
 });
+
```

Comment:
I'm having an issue now, after doing a few pages, of this file and `reducerMethods` getting a bit unrulely. I think the next thing (after what I have written is merged) would be to create some groupings. Would be interested in thoughts.

Should thunks be separated, btw?
At the moment I am thinking:

- sorting/filtering/focus (not sure of a name for this)
- editing?
- modals

Reply:
Thunks should probably be separate. But ye look at after this